packages/react-native-bpk-component-card/src/BpkCard.android.js

```diff
@@ -79,7 +79,11 @@ const BpkCard = props => {
 
   return (
     <View style={style}>
-      <BpkTouchableNativeFeedback accessibilityComponentType="button" {...rest}>
+      <BpkTouchableNativeFeedback
+        borderlessBackground={Platform.Version !== 28}
```

Comment:
Should this condition be checked by `BpkTouchableNativeFeedback` itself so that we don't have to put it on every use?

Reply:
Because some components desire the borderless background behaviour regardless of api version, BpkButtonLink for example:

![button-link](https://user-images.githubusercontent.com/1440090/51399759-9c8e3b00-1b3e-11e9-8cbc-3f39d3df6df9.gif)

Reply:
You could ask - why not just set it to `false`.

I was trying to limit the blast radius of this fix but perhaps when api level 29 comes out it will surface again 🤔

Reply:
Oh ok cool. tbh I assumed that wouldn't be desirable behaviour for any components! My bad!